src/consts.h

```diff
@@ -24,17 +24,46 @@
 #define SLIMEVR_CONSTS_H_
 
 // List of constants used in other places
-#define IMU_UNKNOWN 0
-#define IMU_MPU9250 1
-#define IMU_MPU6500 2
-#define IMU_BNO080 3
-#define IMU_BNO085 4
-#define IMU_BNO055 5
-#define IMU_MPU6050 6
-#define IMU_BNO086 7
-#define IMU_BMI160 8
-#define IMU_ICM20948 9
-#define IMU_ICM42688 10
+
+enum class ImuID {
+    Unknown = 0,
+    MPU9250,
+    MPU6500,
+    BNO080,
+    BNO085,
+    BNO055,
+    MPU6050,
+    BNO086,
+    BMI160,
+    ICM20948,
+    ICM42688,
+    BMI270,
+    LSM6DS3TRC,
+    LSM6DSV,
+    MPU6050_SF,
```

Comment:
I don't have a strong feeling about that, I added it as I thought it would be nice to differentiate MPU "mode" on server.
I can make both use MPU6050 ID if someone also agrees.

Reply:
Well i agree on that it should be visible on the server somehow. But misusing the Field for the IMU i dont rly like.
I think more like a new Field for the Fusion that is used would be better. 
{"INTERNAL-6DOF",
 "INTERNAL-9DOF",
 "VQF-6DOF",
 "VQF-9DOF",
 ...}

Reply:
After reconsideration I changed sfusion MPU6050 driver to return just MPU6050, removed MPU6050_SF entry. Define to select it needs to stay, but returned value remain consistent.

Reply:
After reconsideration I changed sfusion MPU6050 driver to return just MPU6050, removed MPU6050_SF entry. Define to select it needs to stay, but returned value remain consistent.